pyproject.toml

```diff
@@ -28,28 +28,28 @@ python = ">=3.9,<3.12"
 jsonpath-ng = "^1.5.3"
 argparse = "^1.4.0"
 lxml = "^4.9.2"
-requests = "^2.28.1"
+requests = "^2.30.0"
 jsonschema = "^4.17.3"
-openpyxl = "^3.0.10"
-networkx = "^2.8.8"
-pandas = "^1.5.2"
-xlrd = ">=1.0.0"            # without this, pandas raises an ImportError: Missing optional dependency 'xlrd'
-regex = "^2022.10.31"
+openpyxl = "^3.1.2"
+networkx = "^3.1.0"
+pandas = { version = "^2.0.1", extras = ["excel"] }  # extra package that contains xlrd that is necessary for reading old .xls Excel files
+regex = "^2023.5.5"
+docker = "^6.1.2"
```

Comment:
Docker is necessary in the Fast xmlupload PR.
I added docker already now, so that I can remove all changes of `pyproject.toml` and `poetry.lock` from the Fast xmlupload PR.
This way, I won't get any merge conflicts.

Reply:
just wondering: if you have this docker dependency here - could this also be used for the docker based DSP-Tools commands like starting the stack?

Reply:
After a quick look into `src/dsp_tools/utils/stack_handling.py`, I only see one possibility: There are 4 lines of code where a docker command is executed via `subprocess.run("docker do something")`. This could perhaps be replaced by the docker library.
But apart from that, I don't see a significant potential for this library.
But thank you for giving me the hint - it's always good to reflect about potential improvements :-)